components/inputs/input-text.js

```diff
@@ -135,10 +225,19 @@ class InputText extends RtlMixin(LitElement) {
 
 	_handleInput(e) {
 		this.value = e.target.value;
+		this.dispatchEvent(new CustomEvent('d2l-input-container-input', { bubbles: true, composed: true }));
```

Comment:
What's this needed for? Previously I've wired up to the `input` event and things have worked...

Reply:
I think it was something I had added when working on `input-date`. I'll remove for now and try to work without it.